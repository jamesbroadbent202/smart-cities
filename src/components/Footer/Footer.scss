@import '~style/variables';

.wrapper {
  border-top: 6px solid $color-primary--700;
  padding: 25px 0;
}
