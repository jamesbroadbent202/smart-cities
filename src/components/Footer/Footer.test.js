import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

it('should match Snapshot', () => {
  const renderedValue =  renderer.create(<Footer />).toJSON();
  expect(renderedValue).toMatchSnapshot();
});
