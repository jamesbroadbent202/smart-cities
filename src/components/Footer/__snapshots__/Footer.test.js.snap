// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match Snapshot 1`] = `
<footer
  className="wrapper"
>
  <h1>
    Footer
  </h1>
</footer>
`;
